services/movie_session.py

```diff
@@ -0,0 +1,43 @@
+from datetime import datetime
+from typing import Optional
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
+    )
+
+
+def get_movies_sessions(session_date: datetime = None) -> MovieSession:
```

Comment:
invalid return typehint
+
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]

Reply:
not fixed

Reply:
still not fixed :) MoviesSession is not being returned by this function